measure

```diff
@@ -247,48 +380,102 @@ class NewRelicDriver(Measure):
         self.start_timer()
 
     def measure(self):
-        skip_sleep = os.environ.get('SKIP_SLEEP', False)
-        self.progress_message = 'Measurements started'
-        self.print_progress()
+        sleep_fn = time.sleep # overriden when slo provided
+        skip_sleep = os.environ.get('SKIP_SLEEP', False) # NOTE: this skips SLO checking as well as sleeps
```

Comment:
I assume that 'SKIP_SLEEP' is used only in debugging/testing, the code doesn't seem to be operable without it for any normal measurement. I recommend adding a comment that this is a debug-only setting (or, renaming the setting to something like DEBUG_SKIP_SLEEP might help, in lieu of an explanatory comment).

Reply:
Renamed skip_sleep to DEBUG_SKIP_SLEEP, moved to the top for visibility and capitalized to reflect the fact its not changed. Added a comment to indicate it should not be used only for debug and not in optimizations